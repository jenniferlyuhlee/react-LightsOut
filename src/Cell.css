.Cell {
  height: 100px;
  width: 100px;
  background-color: darkgray;
  cursor: pointer;
  border: 2px solid black;
  margin: 0px;
}

.Cell-lit {
  background-color: greenyellow;
}
