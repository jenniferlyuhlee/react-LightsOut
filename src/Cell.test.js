import React from "react";
import {render} from "@testing-library/react";
import Cell from "./Cell";

// smoke test
test("it renders without crashing", () => {
    render (
        <table>
            <tbody>
                <tr>
                    <Cell isLit={false} />
                </tr>
            </tbody>
        </table>
    )
})

// snapshot test
test("matches snapshot", () => {
    const {asFragment} = render(        
                <table>
                    <tbody>
                        <tr>
                            <Cell isLit={false} />
                        </tr>
                    </tbody>
                </table>);
    expect(asFragment()).toMatchSnapshot();
})