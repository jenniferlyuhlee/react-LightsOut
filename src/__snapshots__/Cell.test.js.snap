// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="Cell "
          role="button"
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
